src/doppkit_toolbox/esri/toolboxes/doppkit_wrapper_toolbox.pyt

```diff
@@ -77,26 +87,49 @@ class FetchExport:
             parameterType="Required",
             direction="Input",
         )
+
         add_to_map.value = True
+
         return [grid_server, grid_access_token, aoi_name, dl_directory, add_to_map]
 
     def updateParameters(self, parameters):
```

Comment:
Try removing method altogether, if that doesn't work, remove the commented content.